spec/ruby/core/module/fixtures/autoload_during_require_current_file.rb

```diff
@@ -0,0 +1,5 @@
+module ModuleSpecs::Autoload
+  autoload(:Foo_1596304340, __FILE__)
```

Comment:
We don't need a random-looking name since it's scoped under `ModuleSpecs::Autoload`.
How about `AutoloadCurrentFile` or so?